pages/en/lb4/readmes/example-getting-started.md

```diff
@@ -0,0 +1,424 @@
+  " %}
+
+Let's get started building our very first application using LoopBack 4!
+We're going to build a Todo application that provides a CRUD API for Todo lists!
+
+## Prerequisites
+
+Before we can begin, you'll need to make sure you have some things installed:
+- [Node.js](https://nodejs.org/en/) at v6.x or greater
+
+Additionally, this tutorial assumes that you are comfortable with
+certain technologies, languages and concepts.
+- JavaScript (ES6)
+- [npm](https://www.npmjs.com/)
+- [REST](https://en.wikipedia.org/wiki/Representational_state_transfer)
+
+## The Final Result
+
+If you'd like to see the final result of what you'll be making during this
+tutorial, you can find the app in the
+[example-getting-started](https://github.com/strongloop/loopback-next/tree/master/packages/example-getting-started)
+package in the [loopback-next monorepo](https://github.com/strongloop/loopback-next/tree/master).
+
+## Tutorial
+
+Here's a step-by-step guide of how to build this repository!
+
+### Create your app scaffolding
+
+Install the `@loopback/cli` package. This will give you the command-line
+toolkit that can generate a basic REST app for you.
+
+```
+npm i -g @loopback/cli
+```
+
+Next, navigate to whichever directory you'd like to create your new project
+and run `lb4`. Follow the prompts to generate your application. For this
+tutorial, when prompted with the options for selecting things like whether or
+not to enable certain project features (loopback's build, tslint, mocha, etc.),
+leave them all enabled.
+
+<!-- TODO: Add screenshot of terminal here to illustrate what we mean. -->
+
+### Adding Legacy Juggler Capabilities
+
+Jump into the directory for your new application. You'll see a folder structure
+similar to this:
+
+```
+dist\
+node_modules\
+src\
+  controllers\
+    ping.controller.ts
+    README.md
+  repositories\
+    README.md
+  application.ts
+  index.ts
+test\
+  mocha.opts
+  ping.controller.test.ts
+  README.md
+index.js
+index.d.ts
+index.ts
+```
+
+The application template comes with a controller, and some default wireup in
+`src/application.ts` that handles the basic configuration for your application.
+For this tutorial, we won't need `ping.controller.ts` or its corresponding test,
+but you can leave them in for now.
+
+Now that you have your setup, it's time to modify it to add in
+`@loopback/repository`. Install this dependency by running this command:
+
+```
+npm i --save @loopback/repository
+```
+
+Next, modify `src/application.ts` to change the base class of your app to use
+the `RepositoryMixin`:
+
+{% include code-caption.html content="src/application.ts" %}
+
+```ts
+import {Application, ApplicationConfig} from '@loopback/core';
+import {RestComponent} from '@loopback/rest';
+import {PingController} from './controllers/ping-controller';
+import {Class, Repository, RepositoryMixin} from '@loopback/repository';
+
+export class TodoApplication extends RepositoryMixin(Application) {
```

Comment:
Could you please refactor the tutorial to use `RestApplication`? Refer to the latest `master` branch for consistency.

Reply:
I've updated the README in this PR #961, so you can just copy paste the diffs